minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -1013,6 +1027,17 @@ public synchronized void stop() throws IOException, InterruptedException {
       p.waitFor();
     }
     miniDFS.set(null);
+    clusterState = State.STOPPED;
+  }
+
+  @Override
+  public void terminate() {
```

Comment:
could synchronize this method like the start() and stop() methods.

Reply:
Implemented in b5a3214